source/Hardpoint.cpp

```diff
@@ -357,6 +409,18 @@ bool Hardpoint::FireSpecialSystem(Ship &ship, const Body &body, std::vector<Visu
 	if(offset.Length() > range)
 		return false;
 
+	// Check if the target is within the arc of fire.
+	Angle aim(offset);
```

Comment:
This declaration causes the second one (line 429) to fail. You need it here, so remove the other one.

Reply:
Fixed in [608c8b7](https://github.com/endless-sky/endless-sky/pull/7094/commits/608c8b7195892d345a056c12ceb3cad6795da54a). Thanks for spotting it while I still can't get the CMake thing to work!